### DISCUSSION TITLE
Getting stuck forever in build phase

### DISCUSSION THREAD

**Discussion by JasonKramer525** [vercel/vercel#5003] [Help]
I am trying to deploy a FastAPI REST server that I made. I have a requirements.txt file with all of my dependencies, and am using uvicorn.

I am using gunicorn and my build command is "gunicorn -k uvicorn.workers.UvicornWorker main:app". The console prints "Application startup complete" but then nothing occurs afterward and it ends up stuck on the build phase.

`00:09:22.879 [2020-08-07 07:09:22 +0000] [1394] [INFO] Starting gunicorn 20.0.4 00:09:22.879 [2020-08-07 07:09:22 +0000] [1394] [INFO] Listening at: http://127.0.0.1:8000 (1394) 00:09:22.879 [2020-08-07 07:09:22 +0000] [1394] [INFO] Using worker: uvicorn.workers.UvicornWorker 00:09:22.881 [2020-08-07 07:09:22 +0000] [1397] [INFO] Booting worker with pid: 1397 00:09:23.017 [2020-08-07 07:09:23 +0000] [1397] [INFO] Started server process [1397] 00:09:23.017 [2020-08-07 07:09:23 +0000] [1397] [INFO] Waiting for application startup. 00:09:23.017 [2020-08-07 07:09:23 +0000] [1397] [INFO] Application startup complete.`

**Comment by Kamrulhasan12345** [comment]
Maybe that is because the application is loading in the localhost only. Try to change the host to 0.0.0.0 with `--bind=0.0.0.0` placeholder
